karapace/protobuf/schema.py

```diff
@@ -1,14 +1,168 @@
 # Ported from square/wire:
 # wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Schema.kt
 # Ported partially for required functionality.
+from karapace.protobuf.enum_element import EnumElement
+from karapace.protobuf.location import Location
+from karapace.protobuf.message_element import MessageElement
+from karapace.protobuf.option_element import OptionElement
+from karapace.protobuf.proto_file_element import ProtoFileElement
+from karapace.protobuf.proto_parser import ProtoParser
+from karapace.protobuf.utils import append_documentation, append_indented
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def add_slashes(text: str) -> str:
+    escape_dict = {
+        '\a': '\\a',
+        '\b': '\\b',
+        '\f': '\\f',
+        '\n': '\\n',
+        '\r': '\\r',
+        '\t': '\\t',
+        '\v': '\\v',
+        '\'': "\\'",
+        '\"': '\\"',
+        '\\': '\\\\'
+    }
+    result: str = ""
+    for char in text:
+        try:
+            result += escape_dict[char]
+        except KeyError:
+            result += char
```

Comment:
based on https://stackoverflow.com/questions/2522005/cost-of-exception-handlers-in-python this might be an expensive operation compared to `if` since we are expecting that most of the text will not need to be escaped. Not a blocker for me, but are we sure we want this style?

Reply:
Yes we can get rid of try except when go to if statement. 
We can do long if elif elif statement or use following construction I like it because it is shorter with relatively the same performance. 
`    for char in text:
        c = escape_dict.get(char)
        result += c if c is not None else char
    return result
`

Reply:
If you're worried about performance, please use the standard library implementation:

https://github.com/instaclustr/karapace/pull/12#discussion_r707650817